extensions/BUILD.gn

```diff
@@ -1,6 +1,4 @@
-if (is_android || is_ios) {
```

Comment:
@bridiver I update this PR before reading your links. But, I understood that is_ios should be removed here. But, there is a build error when I just removed it here as below,

```

10:09:27  ERROR Unresolved dependencies.
10:09:27  //brave/common:shield_exceptions(//build/toolchain/mac:ios_clang_x64)
10:09:27    needs //brave/extensions:common(//build/toolchain/mac:ios_clang_x64)
10:09:27  
10:09:27  null
10:09:27  null
``` 

So, I think this PR needs to make //brave/common:shield_exceptions not have "//brave/extensions:common". Does it make sense?